src/Features/VisualBasic/Portable/CodeRefactorings/ConvertIfToSwitch/VisualBasicConvertIfToSwitchCodeRefactoringProvider.Pattern.vb

```diff
@@ -0,0 +1,71 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 9.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.ConvertIfToSwitch
+    Partial Class VisualBasicConvertIfToSwitchCodeRefactoringProvider
+        Private MustInherit Class Pattern
```

Comment:
hrmm... it still feels like it would be nice to have it to force new subclasses to fit this 'pattern' (no pun intended).